charts/partner-onboarder/values.yaml

```diff
@@ -443,10 +443,15 @@ onboarding:
       enabled: false
     - name: mimoto-keybinding
       enabled: true
+    - name: mimoto-oidc
+      enabled: false
 
   configmaps:
     s3:
       s3-host: 'http://minio.minio:9000'
       s3-user-key: 'admin'
       s3-region: ''
+    onboarder-namespace:
```

Comment:
onboarder-namespace?

Reply:
As per Syed's suggestion to pass the namespace values to the default.sh of dockerfile through helm chart, I have added these changes